Job launching after 0.27 seconds in submission.
Running python job.
Changed into dir /Users/weisser/Dropbox (MIT)/MIT/Research/MLTools/Dalitz_simplified/spearmint
Importing classifier_eval_callable.py
Running classifier_eval_callable.main()
Traceback (most recent call last):
  File "/Users/weisser/Documents/Spearmint-master/spearmint/launcher.py", line 243, in launch
    result = python_launcher(job)
  File "/Users/weisser/Documents/Spearmint-master/spearmint/launcher.py", line 327, in python_launcher
    result = module.main(job['id'], params)
  File "/Users/weisser/Dropbox (MIT)/MIT/Research/MLTools/Dalitz_simplified/spearmint/classifier_eval_callable.py", line 9, in main
    return classifier_eval_simplified(params['aC'], params['agamma'])
TypeError: 'module' object is not callable
Problem executing the function
Job failed in 0.66 seconds.

This script can be used to get the p value for classifiers. It takes input files with column vectors corresponding to features and lables. 
Then there are two different routes one can go down. When optimise_hyperparam_mode is enabled (1), then a grid search will be performed on 
one set of input files. When the mode is turned off (0), then the p value is computed for multiple sets of input files and the p value 
distribution is plotted.  

Anything printed here will end up in the output directory for job #2
{u'aC': array([ 0.1]), u'agamma': array([  1.00000000e-07])}
(<type 'exceptions.TypeError'>, TypeError("'module' object is not callable",), <traceback object at 0x112508fc8>)
